src/data.js

```diff
@@ -57,114 +54,139 @@ class Data {
         domain_specific_data
       )
       VALUES($1, $2, $3, $4, to_timestamp($5), to_timestamp($5), $6, ST_PointFromText($7, 4326), $8, $9, $10, $11, $12, $13, $14 ) RETURNING *`,
-      values: [planterId, lat, lon, gpsAccuracy, timestamp, imageUrl, geometry, planterPhotoUrl, planterIdentifier, deviceIdentifier, note, uuid, JSON.stringify(imagesJson), JSON.stringify(domainSpecificDataJson) ],
+      values: [
+        planterId,
+        lat,
+        lon,
+        gpsAccuracy,
+        timestamp,
+        imageUrl,
+        geometry,
+        planterPhotoUrl,
+        planterIdentifier,
+        deviceIdentifier,
+        note,
+        uuid,
+        JSON.stringify(imagesJson),
+        JSON.stringify(domainSpecificDataJson),
+      ],
     }
 
-    const rval = await this.pool.query(insertQuery)
-      .catch(err => {
-        console.log(`ERROR: FAILED TO CREATE TREE ${err}`)
-        console.log(insertQuery);
-        throw(err);
-      });
-    const tree = rval.rows[0];
-
-    tree.attributes = await this.insertTreeAttributes(tree.id, body.attributes);
+    const rval = await this.pool.query(insertQuery).catch(err => {
+      console.log(`ERROR: FAILED TO CREATE TREE ${err}`)
+      console.log(insertQuery)
+      throw err
+    })
+    const tree = rval.rows[0]
 
-    return tree;
+    tree.attributes = await this.insertTreeAttributes(tree.id, body.attributes)
 
+    return tree
   }
 
-  async insertTreeAttributes(id, attributes){
-    var results = [];
-    if(attributes) {
-      for(let attribute of attributes){
+  async insertTreeAttributes(id, attributes) {
+    var results = []
+    if (attributes) {
+      for (let attribute of attributes) {
         const insert = {
           text: `INSERT INTO tree_attributes
           (tree_id, key, value)
           values
           ($1, $2, $3)
           RETURNING *`,
-          values: [id, attribute.key, attribute.value]
+          values: [id, attribute.key, attribute.value],
         }
-        const stored = await this.pool.query(insert);
```

Comment:
Isn't semicolons recommended even though it is optional?

Reply:
You are absolutely correct! I just realized that this project did not have a Prettier config file. I made changes to add the config file and include semicolons. Thanks!